embedded-service/src/type_c/comms.rs

```diff
@@ -11,3 +11,23 @@ pub struct DebugAccessoryMessage {
     /// Connected
     pub connected: bool,
 }
+
+/// UCSI connector change message
+#[derive(Copy, Clone, Debug)]
+#[cfg_attr(feature = "defmt", derive(defmt::Format))]
+pub struct UcsiConnectorChange {
```

Comment:
You mentioned the spec calls this a "change indicator." Can we use the same name?